File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusToolDefinition.kt
package com.sdkwork.backend

data class PlusToolDefinition(
    val id: Int? = null,
    val uuid: String? = null,
    val type: String? = null,
    val name: String? = null,
    val description: String? = null,
    val inputSchema: String? = null,
    val buildIn: Boolean? = null,
    val httpDefinition: HttpApiDefinition? = null
)

File: sdkwork-backend-sdk-typescript/src/types/plus-tool-definition.ts
import type { HttpApiDefinition } from './http-api-definition';

export interface PlusToolDefinition {
  id?: number;
  uuid?: string;
  type?: 'HTTP' | 'TOOL' | 'MCP' | 'SCRIPT';
  name?: string;
  description?: string;
  inputSchema?: string;
  buildIn?: boolean;
  httpDefinition?: HttpApiDefinition;
}
